dorpsgek_github/config.py

```diff
@@ -1,8 +1,13 @@
-import os
+REFERENCE_REPOSITORY_FOLDER = "/var/lib/dorpsgek/reference"
+WORKING_FOLDER = "/var/lib/dorpsgek-github/workdir"
 
-GITHUB_APP_ID = os.environ["GITHUB_APP_ID"]
-GITHUB_APP_SECRET = os.environ["GITHUB_APP_SECRET"]
-GITHUB_APP_PORT = os.getenv("GITHUB_APP_PORT", 8080)
+GITHUB_APP_ID = None
+GITHUB_APP_SECRET = None
+GITHUB_APP_PRIVATE_KEY_FILE = None
+GITHUB_APP_PRIVATE_KEY = None
 
-with open(os.environ["GITHUB_APP_PRIVATE_KEY_FILE"]) as f:
-    GITHUB_APP_PRIVATE_KEY = f.read()
+GITHUB_APP_PORT = 8080
+RUNNER_PORT = 8081
+
+MODULES = ""
+DORPSGEK_COMMANDS = ""
```

Comment:
the format of these isn't mentioned anywhere - it calls split() on them, but that's not clear from just the config file..